controllers/replication.storage/volumereplication_controller.go

```diff
@@ -158,6 +164,20 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		}
 
 		volumeHandle = pv.Spec.CSI.VolumeHandle
+		logger.Info("volume handle", "VolumeHandleName", volumeHandle)
+	case volumeGroupReplicationDataSource:
+		vgrc, vgrErr = r.getVolumeGroupReplicationContent(logger, nameSpacedName)
+		if vgrErr != nil {
+			logger.Error(vgrErr, "failed to get VolumeGroupReplication", "VGRName", instance.Spec.DataSource.Name)
```

Comment:
```suggestion
			logger.Error(vgrErr, "failed to get VolumeGroupReplicationContent", "VGRName", instance.Spec.DataSource.Name)
```

Reply:
we cannot exactly tell what is missing VolumeGroupReplicationContent or VolumeGroupReplication, IMO we should keep VolumeGroupReplication similar to PVC error above